### DISCUSSION TITLE
Cant enter github username

### DISCUSSION THREAD

**Discussion by skorphil** [jupyterlab/jupyterlab-git#1017] [Q&A]
Hi, i'm running JupiterLab on remote Ubuntu server.
I'm launching it with
```
 jupyter lab --no-browser --port=8888 &
```

Then i launch SSH tunnel on my local machine and here we go...

After i installed Git extension, the jupiter server started to ask me for my username after i launching jupiter lab server:
```
Username for 'https://github.com':
```

So i entered my username and got an error
```
Username for 'https://github.com': skorphil
skorphil: command not found

[1]+  Stopped                 jupyter lab --no-browser --port=8888
```

After this jupiter server is crashing, i cant turn it off using `jupyter server stop 8888` and have to reboot my whole server.
This all hapens every time i'm trying to enter my github username

The question - how do i enter my username and finally will be able to use github extension?

**Comment by skorphil** [comment]
I got it. Just cloned my random private repo
```
git clone <repo>
```

then enter username and token... now git saved em and it's not asking me username when i launch jupiter server

**Comment by fcollonval** [comment]
Small suggestion, if you use SSH authentication you can make use of SSH agent forwarding: https://docs.github.com/en/developers/overview/using-ssh-agent-forwarding
